src/chrome/content/rules/IEEE.xml

```diff
@@ -130,14 +113,6 @@
 		<test url="http://www.ieee.org/go/shop" />
 		<test url="http://www.ieee.org/go/shop/" />
```

Comment:
I think we can also remove these extra rules

Reply:
Yeah those URLs seem to behave better now.